oci/private/tarball.sh.tpl

```diff
@@ -101,15 +98,11 @@ for LAYER in $(${JQ} -r ".[]" <<< $LAYERS); do
   add_to_tar "${IMAGE_DIR}/blobs/${LAYER}" "blobs/${LAYER}.tar.gz"
 done
 
-manifest_json=$(mktemp)
```

Comment:
as above, `mktemp` doesn't work now that the work is split into two actions.
This time we make it an explicit output of this action, and input to the `tar` action.